flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -452,29 +493,15 @@ under the License.
 						<configuration>
 							<shadeTestJar>false</shadeTestJar>
 							<artifactSet>
-								<!-- Needs end-to-end tests to ensure the built flink-connector-hive jar contains all required dependencies and can run -->
 								<includes>
-									<include>commons-dbcp:commons-dbcp</include>
-									<include>commons-pool:commons-pool</include>
-									<include>commons-beanutils:commons-beanutils</include>
-									<include>com.fasterxml.jackson.core:*</include>
-									<include>com.jolbox:bonecp</include>
-									<include>org.apache.thrift:libthrift</include>
-									<include>org.datanucleus:*</include>
-									<include>org.antlr:antlr-runtime</include>
+									<include>*:*</include>
 								</includes>
 							</artifactSet>
 							<promoteTransitiveDependencies>true</promoteTransitiveDependencies>
 							<!-- DO NOT RELOCATE GUAVA IN THIS PACKAGE -->
 							<filters>
 								<filter>
-									<!-- some dependencies bring their own LICENSE.txt which we don't need -->
 									<artifact>*:*</artifact>
-									<excludes>
```

Comment:
How come we don't need those any more?

Reply:
because we don't package any hive/hadoop dependencies anymore a long time ago. They are all provided, so we don't need to filter anything for them. This is a kind of historical leftover, and can be on its own PR if necessary.

Reply:
This is fine.